src/styles/ia-button.ts

```diff
@@ -0,0 +1,107 @@
+import { css } from 'lit';
```

Comment:
lol let's package this up 🎁

Reply:
Yeah, we are using this single file in multiple repo and sometimes make specific changes in this file per requirements leads to inconsistency in this file. we should publish as a package so we can have atleast common behaviour on this file.